sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -1940,14 +1941,14 @@ def message_content():
                 for i in range(20):
                     yield ServiceBusMessage(
                         body="Test message",
-                        application_properties={'key': 'value'},
+                        # application_properties={'key': 'value'},
                         subject='1st',
-                        content_type='application/text',
-                        correlation_id='cid',
-                        message_id='mid',
-                        to='to',
-                        reply_to='reply_to',
-                        time_to_live=timedelta(seconds=60)
+                        # content_type='application/text',
+                        # correlation_id='cid',
+                        # message_id='mid',
+                        # to='to',
+                        # reply_to='reply_to',
+                        # time_to_live=timedelta(seconds=60)
```

Comment:
Do we have some way to track these commented tests after the pyamqp/uamqp switch work?

Reply:
I can make an issue for them